pages/api/GameApi.jsx

```diff
@@ -0,0 +1,102 @@
+import CONSTANTS from "../../const";
+
+const gameApi = {
```

Comment:
Why do we need this? Why aren't we calling the symfony API directly from the client app? Or am I missing something?